frontend/packages/shared/src/components/GiteaHeader/ThreeDotsMenu/LocalChangesModal/DeleteModal/DeleteModal.tsx

```diff
@@ -58,29 +70,35 @@ export const DeleteModal = ({
       }
     >
       <div className={classes.contentWrapper}>
-        <Paragraph size='small' spacing>
-          {t('local_changes.modal_delete_modal_text')}
-        </Paragraph>
-        <Textfield
-          label={t('local_changes.modal_delete_modal_textfield_label')}
-          size='small'
-          value={nameToDelete}
-          onChange={(e) => setNameToDelete(e.target.value)}
-        />
-        <div className={classes.buttonWrapper}>
-          <Button
-            variant='secondary'
-            color='danger'
-            onClick={handleDelete}
-            disabled={appName !== nameToDelete}
-            size='small'
-          >
-            {t('local_changes.modal_confirm_delete_button')}
-          </Button>
-          <Button variant='secondary' onClick={handleClose} size='small'>
-            {t('general.cancel')}
-          </Button>
-        </div>
+        {isLoading ? (
+          <StudioSpinner />
+        ) : (
+          <>
+            <Paragraph size='small' spacing>
+              {t('local_changes.modal_delete_modal_text')}
+            </Paragraph>
+            <Textfield
+              label={t('local_changes.modal_delete_modal_textfield_label')}
```

Comment:
Should we add the name of the app within this label to make it easier for the users to know the app name? Today you need to remember the app name or check it within the URL. 😊
```suggestion
              label={t('local_changes.modal_delete_modal_textfield_label', {
                 appName: app
              })}
```

And then the result could be: Skriv inn navnet på appen for å bekrefte. Appens navn er: {{appName}}